test/e2e/network/loadbalancer.go

```diff
@@ -1236,6 +1244,24 @@ var _ = common.SIGDescribe("LoadBalancers", func() {
 			framework.Failf("Failed to connect to backend 2")
 		}
 	})
+
+	ginkgo.It("should not have connectivity disruption during rolling update with externalTrafficPolicy=Cluster [Slow]", func() {
+		// We start with a low but reasonable threshold to analyze the results.
+		// The goal is to achieve 99% minimum success rate.
+		// TODO: We should do incremental steps toward the goal.
+		minSuccessRate := 0.95
```

Comment:
Based on success rate in [here](https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/114052/pull-kubernetes-e2e-gci-gce-ingress/1599731968268832768/build-log.txt), I'm still inclined to make this value higher, but I think we can increase this in a follow-up after we have some some CI run data